lix/packages/sdk/src/newLix.ts

```diff
@@ -34,15 +40,19 @@ export async function newLixFile(): Promise<Blob> {
         operation TEXT NOT NULL,
         value TEXT,
         meta TEXT,
-        commit_id TEXT
-        ) strict;
+        commit_id TEXT,
+        conflict TEXT
```

Comment:
Conflict is a reference to another change id right? 

If so, I would expect `conflict_with_change_id` or `conflict_change_id`

Reply:
removed the conflict until i the pr for merging.